kyo-core/shared/src/main/scala/kyo/Adder.scala

```diff
@@ -71,28 +90,29 @@ class DoubleAdder private[kyo] (private val ref: j.DoubleAdder) extends AnyVal:
       * @return
       *   Unit
       */
-    inline def add(v: Double)(using Frame): Unit < IO = IO(ref.add(v))
+    inline def add(v: Double)(using inline frame: Frame): Unit < IO = IO(unsafe.add(v))
```

Comment:
minor cleanup: there was no reason not to inline `frame`, which avoids an extra field in the `IO` suspension.